usagov/vote-gov-drupal
# VOTE-2764: Implement loading feature on slow page load

[VOTE-2764](https://cm-jira.usa.gov/browse/VOTE-2764)

Insert a link to the Jira ticket (e.g VOTE-XXX).

## Description

Adds an animated loading gif in the NVRF root div while the content loads on slower speeds

### QA/Testing instructions

1. lando retune
2. npm run build
3. Navigate to http://vote-gov.lndo.site/register/alaska/mail-in-form-filler
4. Using the network tab in dev tools, change throttle speed
5. Confirm the gif loads before content and is positioned center of screen

## Checklist for the Developer




## Checklist for the Peer Reviewers
